x-pack/legacy/plugins/siem/public/pages/case/components/case_view/index.tsx

```diff
@@ -192,15 +195,27 @@ export const CaseComponent = React.memo<CaseProps>(({ caseId, initialData }) =>
                       </EuiBadge>
                     </EuiDescriptionListDescription>
                   </EuiFlexItem>
-                  <EuiFlexItem>
-                    <EuiDescriptionListTitle>{i18n.CASE_OPENED}</EuiDescriptionListTitle>
-                    <EuiDescriptionListDescription>
-                      <FormattedRelativePreferenceDate
-                        data-test-subj="case-view-createdAt"
-                        value={caseData.createdAt}
-                      />
-                    </EuiDescriptionListDescription>
-                  </EuiFlexItem>
+                  {caseData.status === 'open' ? (
```

Comment:
I think is better to create a new component where you pass it the status (open or close) as a prop and it renders accordingly. This way the duplicated code will be eliminated. What do you think?

Reply:
easy enough!